ci/ubuntu.dockerfile

```diff
@@ -7,20 +7,37 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+
+ RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
+ RUN add-apt-repository "deb https://repo.sovrin.org/ursa/deb xenial master"
+
 RUN apt-get update -y && apt-get install -y \
     python3-nacl \
-    libindy-crypto=0.4.5 \
+
+    cmake \
+    autoconf \
+    libtool \
+    pkg-config \
+    libssl-dev \
     libindy=1.11.1~1343 \
 # rocksdb python wrapper
     libbz2-dev \
     zlib1g-dev \
     liblz4-dev \
     libsnappy-dev \
+    ursa \
```

Comment:
I think currently indy-crypto python wrapper installs indy-crypto library automatically. Should we do the same for Ursa?

Reply:
Yes I agree it should.